Sources/App/Models/Accessory.swift

```diff
@@ -86,6 +106,8 @@ extension Accessory: NodeRepresentable {
             "product_link": productLink,
             "released": released,
             "date": date,
+            "requireshub": requiresHub,
```

Comment:
Can we change the name of the property to follow the convention with other properties to `required_hub_id` and `requires_hub`? The name of the property in Swift is correct already, just need a change in the seed and when preparing the database.